rewrite-maven/src/test/kotlin/org/openrewrite/maven/RemoveDeadReposTest.kt

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.maven
+
+import org.junit.jupiter.api.Test
+import org.openrewrite.Recipe
+
+class RemoveDeadReposTest : MavenRecipeTest {
```

Comment:
This test should be moved to test/java/org/openrewrite/maven and use the RewriteTest testing harness.